endorser/api/db/migrations/versions/updated_to_support_granular__e6afa1dce289.py

```diff
@@ -0,0 +1,105 @@
+
+"""
+from alembic import op
+import sqlalchemy as sa
+import sqlmodel
+from sqlalchemy.dialects import postgresql
+
+# revision identifiers, used by Alembic.
+revision = "e6afa1dce289"
+down_revision = "d925cb39480e"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table(
+        "allowedcredentialdefinition",
+        sa.Column("allowed_cred_def_id", postgresql.UUID(as_uuid=True), nullable=False),
+        sa.Column(
+            "created_at",
+            postgresql.TIMESTAMP(),
+            server_default=sa.text("now()"),
+            nullable=False,
+        ),
+        sa.Column(
+            "updated_at",
+            postgresql.TIMESTAMP(),
+            server_default=sa.text("now()"),
+            nullable=False,
+        ),
+        sa.Column("issuer_did", sqlmodel.sql.sqltypes.AutoString(), nullable=False),
+        sa.Column("author_did", sqlmodel.sql.sqltypes.AutoString(), nullable=False),
+        sa.Column("schema_name", sqlmodel.sql.sqltypes.AutoString(), nullable=False),
+        sa.Column("version", sqlmodel.sql.sqltypes.AutoString(), nullable=False),
+        sa.Column("tag", sqlmodel.sql.sqltypes.AutoString(), nullable=False),
+        sa.Column("rev_reg_def", sa.Boolean(), nullable=False),
+        sa.Column("rev_reg_entry", sa.Boolean(), nullable=False),
+        sa.PrimaryKeyConstraint("allowed_cred_def_id"),
+    )
+    op.create_table(
+        "allowedpublicdid",
+        sa.Column(
+            "created_at",
+            postgresql.TIMESTAMP(),
+            server_default=sa.text("now()"),
+            nullable=False,
+        ),
+        sa.Column(
+            "updated_at",
+            postgresql.TIMESTAMP(),
+            server_default=sa.text("now()"),
+            nullable=False,
+        ),
+        sa.Column("registered_did", sqlmodel.sql.sqltypes.AutoString(), nullable=False),
+        sa.PrimaryKeyConstraint("registered_did"),
+    )
+    op.create_table(
+        "allowedschema",
+        sa.Column("allowed_schema_id", postgresql.UUID(as_uuid=True), nullable=False),
+        sa.Column(
+            "created_at",
+            postgresql.TIMESTAMP(),
+            server_default=sa.text("now()"),
+            nullable=False,
+        ),
+        sa.Column(
+            "updated_at",
+            postgresql.TIMESTAMP(),
+            server_default=sa.text("now()"),
+            nullable=False,
+        ),
+        sa.Column("author_did", sqlmodel.sql.sqltypes.AutoString(), nullable=False),
+        sa.Column("schema_name", sqlmodel.sql.sqltypes.AutoString(), nullable=False),
+        sa.Column("version", sqlmodel.sql.sqltypes.AutoString(), nullable=False),
+        sa.PrimaryKeyConstraint("allowed_schema_id"),
+    )
+    op.add_column(
+        "endorserequest",
+        sa.Column(
+            "author_goal_code", sqlmodel.sql.sqltypes.AutoString(), nullable=False
+        ),
+    )
```

Comment:
Since this new field is `NOT NULL` it will require a default value.  Otherwise you run into this; https://github.com/bcgov/DITP-DevOps/issues/138#issuecomment-1874093933

Reply:
https://docs.sqlalchemy.org/en/20/core/defaults.html

Reply:
I have corrected this in the code and regenerated the migration